spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/RootController.java

```diff
@@ -62,6 +62,8 @@
 @ExposesResourceFor(RootResource.class)
 public class RootController {
 
+	public static final String BASE_PATH = "/api";
```

Comment:
Maybe not in the scope of this PR. But, we want to be able to make this configurable via some SCDF base path property.